.devcontainer/Dockerfile

```
@@ -1,9 +1,9 @@
-# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.231.5/containers/ubuntu/.devcontainer/base.Dockerfile
+# See here for image contents: https://github.com/devcontainers/images/blob/main/src/base-ubuntu/.devcontainer/Dockerfile
 
-# [Choice] Ubuntu version (use hirsuite or bionic on local arm64/Apple Silicon): hirsute, focal, bionic
-ARG VARIANT="hirsute"
-FROM mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
+# [Choice] Ubuntu version: https://github.com/devcontainers/images/tree/main/src/base-ubuntu
+ARG VARIANT="jammy"
```

```suggestion
ARG VARIANT="ubuntu-22.04"
```

Shall we use the numeric or name based reference here? I personally find the numeric easier to compare at a glance if we are on the latest or not

---

ah you pushed here ... i didnt see :D